modules/ossm-release-notes-3-0-known-issues.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/ossm-release-notes-assembly.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-3-0-known-issues_{context}"]
+= {SMProductName} 3.0 known issues
+
+* link:https://issues.redhat.com/browse/OSSM-8878[OSSM-8878] There is currently a known issue where {ocp-product-title} builder pods fail to create in namespaces with injection enabled.
++
+Workaround: Add injection labels to workloads instead of the namespace when creating `BuildConfigs`.
```

Comment:
```suggestion
* link:https://issues.redhat.com/browse/OSSM-8878[OSSM-8878] In {SMProductName} 3.0, {ocp-product-title} builder pods fail to create in namespaces with injection enabled. As a workaround, add injection labels to workloads instead of the namespace when creating the  `BuildConfigs` resource. 
```

- So far I have seen the above suggested format for known issues. If `workaround:` is something that's been used consistently in service mesh known issues, then you might want to keep that.
- `link:https://issues.redhat.com/browse/OSSM-8878[OSSM-8878] issue description..` also seems like a consistent format. If not, you can add "For more information, see <known_issue_link>" in the end of para.